projboard/tests/test_like.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from projboard.models import Like, User, Article, Subject
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+NAME = "Shalev"
+
+
+@pytest.mark.django_db
+class TestLikeModel:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_subjects(self):
+        sup = Subject(name='N')
+        sup1 = Subject(name='R')
+        sup.save()
+        sup1.save()
+        return [sup, sup1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_user(self):
```

Comment:
Please use for loop here 
You could use 
this dictionary and use loops 

USER_LIST=[{'name':'test1_name', 'email':'tt@ttt.com'},{'name':'test2_name', 'email':'tt2@ttt.com'}]

user_list=[]
for i, user in enumerate(USER_LIST)
   user= User(name=,USER_LIST[i].name nickname=USER_LIST[i].nick_name, email=..., password=....)
   user.save()
   user_list.append(user)
   
  return user_list 
  
  
  You can also receive ideas from here 
  https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#parametrizing-fixtures

Reply:
Hi @Yarboa,
Please take a look at [033bfc9](https://github.com/redhat-beyond/Smarticle/pull/30/commits/033bfc9d62066b95448471aa9a9d1797d788b9ec).
I Changed the fixtures and a some tests